annotations/tests/applicators.json

```diff
@@ -0,0 +1,459 @@
+{
+  "$schema": "../test-suite.schema.json",
+  "description": "The applicator vocabulary",
+  "suite": [
+    {
+      "description": "`properties`, `patternProperties`, and `additionalProperties`",
+      "compatibility": "3",
+      "schema": {
+        "properties": {
+          "foo": {
+            "title": "Foo"
+          }
+        },
+        "patternProperties": {
+          "^a": {
+            "title": "Bar"
+          }
+        },
+        "additionalProperties": {
+          "title": "Baz"
+        }
+      },
+      "tests": [
+        {
+          "instance": {},
+          "assertions": [
+            {
+              "location": "/foo",
+              "keyword": "title",
+              "expected": []
```

Comment:
If `[]` is intended to represent "this annotation should not exist at all", that should be clearly stated in the docs earlier.

Reply:
If so, I think its better omitted?

Reply:
Apparently this is a lot less intuitive than I thought it was. Yes, this is testing that there must not be any annotations at that location. In general, this kind of test is important for making sure annotations are dropped when expected. However, that isn't the case in this test. I'm not sure why I include this specific test.